vpr/src/base/clustered_netlist.h

```diff
@@ -126,9 +126,15 @@ class ClusteredNetlist : public Netlist<ClusterBlockId, ClusterPortId, ClusterPi
         //Returns the physical block
         t_pb* block_pb(const ClusterBlockId id) const;
 
-        //Returns the type of CLB (Logic block, RAM, DSP, etc.)
+        //Returns the type of CLB (Logic block, RAM, DSP, etc.). Calls block_type with boolean set to true.
         t_type_ptr block_type(const ClusterBlockId id) const;
 
+        //Returns the type of CLB. If get_equivalent_if_set is true, if the equivalent tile has been chosen it returns it.
+        t_type_ptr block_type(const ClusterBlockId id, bool get_equivalent_if_set) const;
```

Comment:
Can you add additional comments describing what is going here, I'm confused.  Why is one signature called over the other, what happens if get_equivalent_if_set is true, and what happens if get_equivalent_if_set is false?

Reply:
Done.